lib/neo4j/migration.rb

```diff
@@ -123,5 +126,68 @@ def new_id_for(model)
         end
       end
     end
+
+    class RelabelRelationships < Neo4j::Migration
+      attr_accessor :relationships_filename, :old_format, :new_format
+
+      def initialize(path = default_path)
+        @relationships_filename = File.join(joined_path(path), 'relabel_relationships.yml')
+      end
+
+      MESSAGE = <<MESSAGE
+# Provide relationships which should be relabled.
+# relationships: [students,lessons,teachers,exams]\nrelationships: []
+# Provide old and new label formats:
+# Allowed options are lower_hashtag, lower, or upper
+formats:\n  old: lower_hashtag\n  new: lower
+MESSAGE
+
+      def setup
+        super
+        return if File.file?(relationships_filename)
+        File.open(relationships_filename, 'w') { |f| f.write(MESSAGE) }
+      end
+
+      def migrate
+        config        = YAML.load_file(relationships_filename).to_hash
+        relationships = config['relationships']
+        @old_format   = config['formats']['old']
+        @new_format   = config['formats']['new']
+
+        output 'This task will relabel every given relationship.'
+        output 'It may take a significant amount of time, please be patient.'
+        relationships.each { |relationship| reindex relationship }
+      end
+
+      private
+
+      def count(relationship)
+        Neo4j::Session.query(
```

Comment:
The other migration task (`AddIdProperty`) does just `Neo4j::Session.query`, so I kept that...
In my real-world use on 6.1.10 there was no `.current` either..